lib/dry/schema/message/or/multi_path.rb

```diff
@@ -20,14 +20,14 @@ class MultiPath < Abstract
           def initialize(*args)
             super
             @root = [left, right].flatten.map(&:_path).reduce(:&)
-            @left = left.map { |msg| msg.to_or(root) }
-            @right = right.map { |msg| msg.to_or(root) }
+            @left = left.flatten.map { |msg| msg.to_or(root) }
+            @right = right.flatten.map { |msg| msg.to_or(root) }
```

Comment:
Wouldn't `flat_map` be enough here? `Array#flatten` with no arguments does deep flattening.

Reply:
Each schema nesting level comes with same arrays nesting. I tested it. Without deep flatten it not worked with same error